### ISSUE TITLE
New preact project fails at dom render

### ISSUE THREAD

**Issue by quentin-sommer** [insin/nwb#245]
This issue is a: 

- [x] Bug report
- [ ] Feature request
- [x] Question / support request
- [ ] Other

nwb is installed globally

node  : 7.4.0
npm : 4.1.1
nwb: 0.15.2

node_molues --depth=0
├──nwb@0.15.2
├── preact@7.2.0
└── preact-compat@3.11.0

When creating a react project `nwb new react-app` then installing then `npm run start` everything works fine. However with `nwb new preact-app` then installing then `npm run start` i get this error:
```
file : preact.js:143
Uncaught DOMException: Failed to execute 'createElement' on 'Document': The tag name provided ('[object Object]') is not a valid name.
    at createNode (http://localhost:3000/app.js:2680:171)
    at idiff (http://localhost:3000/app.js:2720:25)
    at diff (http://localhost:3000/app.js:2696:19)
    at render (http://localhost:3000/app.js:2948:16)
    at init (http://localhost:3000/app.js:4833:79)
    at Object../src/index.js (http://localhost:3000/app.js:4836:1)
    at __webpack_require__ (http://localhost:3000/app.js:660:30)
    at fn (http://localhost:3000/app.js:84:20)
    at Object.0 (http://localhost:3000/app.js:4868:18)
    at __webpack_require__ (http://localhost:3000/app.js:660:30)
```

**Comment by insin** [comment]
Looks like this is because we can no longer provide CommonJS interop when using ES modules in Webpack 2.

This change in `index.js` should fix it, will update the app skeleton template with it now:

```diff
 function init() {
-  let App = require('./App')
+  let App = require('./App').default
   root = render(<App/>, document.querySelector('#app'), root)
 }
```

**Comment by quentin-sommer** [comment]
Perfect :)

**Comment by insin** [comment]
Fixed in v0.15.3 - thanks for the bug report!
